addons/mail/static/src/components/chatter_topbar/chatter_topbar.js

```diff
@@ -17,6 +17,7 @@ class ChatterTopbar extends Component {
      */
     constructor(...args) {
         super(...args);
+        this.sprintf = _.str.sprintf;
```

Comment:
Avoid this, compute the string in JS instead.

getter in component in worse case, or computed field in the model best case (in `chatter` in this case I guess).

Reply:
I added a computed field in the thread model so it may depend on "allAttachments" field.